vm/kvm-unit-tests/runtest.sh

```diff
@@ -165,6 +165,38 @@ cat test.log | tee -a $OUTPUTFILE
 # check for any new failures
 grep FAIL test.log >> failures.txt
 
+# Run KVM Unit tests with Advanced Virt (qemu-4.0) if possible
+if dnf repolist --all | grep rhel8-advvirt; then
```

Comment:
The repo is disabled by default, you will need to enable it first, then please disable it after testing.